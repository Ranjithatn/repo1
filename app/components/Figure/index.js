import React from 'react'

function Figure({ is, src }) {
   return (
    <figure className={"image is-" + is}>
      <img src={src} />
    </figure>
   )
}

export default Figure;
