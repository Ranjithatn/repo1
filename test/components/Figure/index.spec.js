import React from "react";
import toJson from "enzyme-to-json";
import Figure from "../../../app/components/Figure/index";
import Adapter from "enzyme-adapter-react-16";
import { configure, shallow, mount } from "enzyme";


configure({ adapter: new Adapter() });


describe('<Figure />', () => {

  it("should render without crashing", () => {
    const component = mount(
      <Figure />
    );
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
