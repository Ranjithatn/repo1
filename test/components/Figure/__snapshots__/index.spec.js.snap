// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Figure /> should render without crashing 1`] = `
<Figure>
  <figure
    className="image is-undefined"
  >
    <img />
  </figure>
</Figure>
`;
